### ISSUE TITLE
Uploading a new package fails with Error: Forbidden

### ISSUE THREAD

**Issue by wereHamster** [haskell/hackage-server#248]
```
$ cabal upload dist/rethinkdb-client-driver-0.0.0.1.tar.gz
Hackage username: wereHamster
Hackage password: 
Uploading dist/rethinkdb-client-driver-0.0.0.1.tar.gz...
Error: dist/rethinkdb-client-driver-0.0.0.1.tar.gz: 403 Forbidden
Error: Forbidden

No access for this resource.
$
```

The password is correct, because when I use the wrong password I get a different error (401 Unauthorized, Error: Username or password incorrect).

This issue is different from #235 in that this package does not exist yet on hackage, it's a completely new package.

**Comment by LukeHoersten** [comment]
@dcoutts how do I contact the hackage admins? I have this problem as well. It's been a big pain for quite a while now for me.

**Comment by redneb** [comment]
@LukeHoersten probably the easiest way is to ask in the #haskell irc channel.

**Comment by gbaz** [comment]
I think this was resolved in #235
